packages/commonwealth/client/scripts/helpers/link.ts

```diff
@@ -18,6 +18,7 @@ export const getLinkType = (link: string): LinkType => {
   if (link.includes('telegram.com')) return 'telegram';
   if (link.includes('t.me')) return 'telegram';
   if (link.includes('twitter.com')) return 'x (twitter)';
+  if (link.includes('x.com')) return 'x (twitter)';
```

Comment:
we could merge line 20 and 21

Reply:
@mzparacha Just made that change, thanks